helm/prometheus/Chart.yaml

```diff
@@ -7,4 +7,4 @@ maintainers:
 name: prometheus
 sources:
   - https://github.com/coreos/prometheus-operator
-version: 0.0.43
\ No newline at end of file
+version: 0.0.44
```

Comment:
I'm not sure if this is the correct approach, but it appears to be needed to pass the tests.